src/components/search-bar/search-bar.css

```diff
@@ -23,7 +23,7 @@ search-bar input {
 search-bar input {
   transition: border 0.3s;
   margin-right: 8px;
-  padding: 10px 16px 10px 56px;
+  padding: 10px 16px 10px 52px;
```

Comment:
The left padding is not divisible by 8. It's made intentionally, cuz the search icon is not symmetric.
It means that focal mass is shifted to the left, so it would look not very accurate if we would have an equal distance from the left and right sides of the icon.